src/components/UploadSection.svelte

```diff
@@ -74,15 +122,18 @@
                             .ogg,
                             .webm"
                     />
-                    <button class="button is-primary is-outlined">Send</button>
+                    <button
+                        class="button is-primary is-outlined"
+                        on:click={sendFiles}>Send</button
+                    >
```

Comment:
Let's initiate upload on `onFileSelected`. The less buttons, the easier it is for user. Less steps to perform an action. I any case I want to upload a file, why do I need one more button to do this?